image/v1/imagev1.go

```diff
@@ -120,6 +120,8 @@ func MakeV1ConfigFromConfig(img *image.Image, v1ID, parentV1ID string, throwaway
 	// Delete fields that didn't exist in old manifest
 	delete(configAsMap, "rootfs")
 	delete(configAsMap, "history")
+	delete(configAsMap, "os_version")
+	delete(configAsMap, "os_features")
```

Comment:
An alternative idea, which may be silly, would be to tag these fields in the actual `Image`/`V1Image` structures, and use reflection to find out which should be excluded.